docs/MAVLINK_PARAMETERS.md

```diff
@@ -0,0 +1,32 @@
+# mLRS Documentation: MAVLink Parameters #
+
+([back to main page](../README.md))
+
+When enabled, the Tx module and receiver can also be configured via MAVLink parameters using a ground control station (GCS) such as MissionPlanner. This can be useful in cases in which the other methods for configuration, i.e., the CLI, Lua script, or OLED, are not (easily) accessible, such as in some stand-alone applications.
+
+Using the MAVLink parameters for configuration has its inconveniences however, due to the limitations of the MAVLink parameter system and support in GCSes.
+
+When enabled, the Tx module presents itself as MAVLink component with system id 51 (like the SiK radios) and component id 68 (= MAV_COMP_ID_TELEMETRY_RADIO). That is, it emits a [HEARTBEAT](https://mavlink.io/en/services/heartbeat.html) MAVLink message which allows a GCS to discover the component, and provides the [parameter microservice](https://mavlink.io/en/services/parameter.html). A GCS can thus connect to this component and read & write its parameters.
```

Comment:
Would suggest combining this with paragraph 1 to avoid have two paragraphs that start with 'When enabled...'